pkgs/by-name/xl/xlink/package.nix

```
@@ -0,0 +1,84 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  pkg-config,
+  libusb1,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xlink";
+  version = "0.0.0-unstable-2025-14-03";
```

```suggestion
  version = "0-unstable-2025-14-03";
```

Usual convention is to use one `0`, inplace of the version if upstream provides no such tags.